rudderstack_blueprints/trigger_sync.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_SYNC_REFRESH_ERROR = 210
+EXIT_CODE_UNKNOWN_ERROR = 211
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--source-id', dest='source_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def trigger_sync(source_id, access_token):
+    api_headers = {
+        'authorization': f"Bearer {access_token}",
+        'Content-Type': 'application/json'
+    }
+    api_url = "https://api.rudderstack.com/v2/sources"
+    trigger_sync_url = api_url + f"/{source_id}/start"
+    # get response from API
+    try:
+        trigger_sync_response = requests.get(trigger_sync_url, 
+                                           headers=api_headers)
+        # check if successful, if not return error message
+        if trigger_sync_response.status_code == requests.codes.ok:
+            print(f"Trigger sync for Source ID: {source_id} successful")
+        else:
+            print(f"Trigger sync failed. Error code:{trigger_sync_response.status_code}")
```

Comment:
I managed to a get a 204, 209, 404, and a 409. Is there any sort of reference as to why these errors would occur so we can let users know?

Reply:
It looks like it https://github.com/rudderlabs/rudder-airflow-provider/blob/main/rudder_airflow_provider/hooks/rudderstack.py

Reply:
500 Looks to be wrong source-id

Reply:
401 appears to be credentials